CMakeLists.txt

```diff
@@ -617,8 +617,16 @@ endif()
 
 set(CLANG_TIDY "" CACHE STRING "CMAKE_CXX_CLANG_TIDY equivalent that only applies to mixxx sources, not bundled dependencies")
 
+if(WIN32)
+  set(NATIVE_SOURCE_FILES src/nativeeventhandlerwin.cpp)
```

Comment:
Until now, we have the concept of adding all files and #ifdef the content out. 
see util/console.cpp and util/mac.cpp 
I am not sure about the implications with different IDEs, but I think we should stick with one concept. 
In case of Eclipse the file is showed grayed out in the #ifdef region.